File: Anya_Seth/2-Pointer/Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 0;
        int count = 0;
        for(int i = 1; i < nums.length ; i++){
            if(nums[i - 1] != nums[i]){
                int temp = nums[i - 1];
                nums[i - 1] = nums[j];
                nums[j] = temp;
                j++;
                count++;
            }
        }
        nums[j] = nums[nums.length - 1];
        count++;
        return count;
    }
}

File: Lakshay Garg/Leetcode/Two_Pointers_Leetcode/Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& v) {
        int n=v.size(),j=1;
        for(int i=1;i<n;i++){
            if(v[i]!=v[i-1]){
                v[j]=v[i];
                j++;
            }
        }
        return j;
    }
};
